pandas/tseries/tests/test_base.py

```diff
@@ -45,6 +45,24 @@ def test_ops_properties_basic(self):
         self.assertEqual(s.day,10)
         self.assertRaises(AttributeError, lambda : s.weekday)
 
+    def test_shift(self):
```

Comment:
@jreback hope I got it right..I moved the test for base(NotImplementedError) to index.